CONTRIBUTING.md

```diff
@@ -1,3 +1,20 @@
+## Steps for Contributing
+1. Create a new branch off of up-to-date master
+2. Make your changes.  If you are adding an endpoint or resource to the API,
+follow "Adding an API Resource"
+3. Make sure your code follows the "Code Standards" section of this document
+4. Use "git rebase -i origin/master" to squash your commits into one or a few. Follow commit message guidelines.
```

Comment:
From my experience it's annoying when a project doesn't list this up front because then you spend time getting your PR together and submit it, only to  be rejected and go back and run rebase and re-write commit messages.  Better to let people know what we expect at the start.